bindata/network/ovn-kubernetes/common/002-rbac-node.yaml

```diff
@@ -173,6 +173,20 @@ rules:
   - patch
   - update
   - watch
+- apiGroups: ["policy.networking.k8s.io"]
```

Comment:
Any specific reason for not putting the RBACs under `OVN_ADMIN_NETWORK_POLICY_ENABLE` if?

Reply:
oh hmm good question. I actually angled to doing this very recently :D forgot to add it to the RBAC stuff. lemme do that